pom.xml

```diff
@@ -176,6 +182,25 @@
             <version>${test.containers.version}</version>
             <scope>test</scope>
         </dependency>
+        <!-- https://mvnrepository.com/artifact/org.apache.zookeeper/zookeeper -->
+        <dependency>
+            <groupId>org.apache.zookeeper</groupId>
+            <artifactId>zookeeper</artifactId>
+            <version>3.4.14</version>
```

Comment:
we should get these fixes from common probably

Reply:
tried to look for an alternative, but it seems like it's just not possible to exclude a dependency that's inherited from the parent. For now, I think our only choice is to pin the version to a clean one, then update common down the road when it's fixed upstream

Reply:
On these release branches common and ES connector are released together. There's no point to fix here only while common is having the affected dependency. 
https://github.com/confluentinc/common/pull/335 
or 
https://github.com/confluentinc/common/pull/331
Which ever lands first.